metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -88,13 +90,26 @@ def __init__(self, *,
                                             encrypted=encrypted,
                                             trust=trust)  # type: Driver
 
-    def is_healthy(self) -> None:
-        # throws if cluster unhealthy or can't connect.  An alternative would be to use one of
-        # the HTTP status endpoints, which might be more specific, but don't implicitly test
-        # our configuration.
-        with self._driver.session() as session:
-            session.read_transaction(self._execute_cypher_query,
-                                     statement='CALL dbms.cluster.overview()', param_dict={})
+    def health(self) -> health_check.HealthCheck:
+        """
+        Runs one or more series of checks on the service. Can also
+        optionally return additional metadata about each check (e.g.
+        latency to database, cpu utilization, etc.).
+        """
+        checks = {}
+        try:
+            # dbms.cluster.overview() is only available for enterprise neo4j users
+            cluster_overview = self._execute_cypher_query(statement='CALL dbms.cluster.overview()', param_dict={})
+            checks = dict(cluster_overview.single())
+            checks['overview_enabled'] = True
+            status = health_check.OK
+        except neobolt.exceptions.ClientError:
+            checks = {'overview_enabled': False}
+            status = health_check.OK  # Can connect to database but plugin is not available
```

Comment:
what plugin is not available?

Reply:
`dbms.cluster.overview()` a few lines above. In the neo4j docs (https://neo4j.com/docs/operations-manual/current/reference/procedures/) it is shown as an enterprise only feature. Executing this procedure using the OSS version raises this client error